pom.xml

```diff
@@ -603,6 +603,46 @@
 
               <exclude>org.apache.parquet.io.api.Binary#get2BytesLittleEndian()</exclude>
               <exclude>org.apache.parquet.schema.LogicalTypeAnnotation$Float16LogicalTypeAnnotation#accept(org.apache.parquet.schema.LogicalTypeAnnotation$LogicalTypeAnnotationVisitor)</exclude>
+              <exclude>org.apache.parquet.filter2.predicate.Operators$Contains</exclude>
```

Comment:
Is it possible to avoid these exclusions?

Reply:
I ended up removing ArrayColumn and rebasing onto the latest master branch, and the exclusions were no longer needed 👍